src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -61,18 +61,18 @@ export class GcsApiService extends GcsService {
   public cancelAll() {
     const itemsToDownload = this.getDownloadingFiles();
     const itemsToUpload = this.getUploadingFiles();
-    const itemsToExport = this.getExportPendingFiles();
+    const itemsToExport = this.getExportingFiles();
     if (itemsToDownload.length > 0) {
```

Comment:
Does this code need to check this here? The same length check is done in `cancelDownloads()` below. Same is true for the other cancels.

Reply:
Change made will only check store's items only once when cancelAll is called.